File: Contains_Duplicate_217.c
int cmp(int *a, int *b){
    return (*a - *b);
}
bool containsDuplicate(int* nums, int numsSize) {
    if(numsSize==1)
        return false;
    qsort(nums, numsSize, sizeof(int), cmp);
    for(int i=0;i<numsSize;i++){
        if (!(nums[i] ^ nums[i+1]))
            return true;
    }
    return false;
}

File: Contains_Duplicate_217.java
public class Solution {
    public boolean containsDuplicate(int[] nums) {/*
        HashMap<Integer,Integer> hash = new HashMap<Integer,Integer>();
        for(int i=0;i<nums.length;i++){
            if(hash.containsKey(nums[i]))
                return true;
            hash.put(nums[i], i);
        
        }
        return false;
        */
        if (nums.length==1)
            return false;
        Arrays.sort(nums);
        for(int i=0;i<nums.length-1;i++){
            if((nums[i]^nums[i+1])==0)
                return true;
            
        }
        return false;
        
    }
}
